TaylorFramework/Modules/scissors/ExtendedComponent.swift

```diff
@@ -56,51 +56,59 @@ final class ExtendedComponent {
         self.components = []
     }
     
-    func appendComponents(components: [ExtendedComponent], array: [SourceKitRepresentable]) -> [ExtendedComponent] {
+    func appendComponents(array: [SourceKitRepresentable], components: NSMutableArray = []) -> [ExtendedComponent] {
         var braces = 0
-        var componentsCopy = components
-        array.enumerate().forEach { (childNumber, structure) in
-            var offsetRange = structure.asDictionary.offsetRange
-            let type = getComponentType(structure.asDictionary.type, bracesCount: braces, isLast: childNumber == array.count - 1)
-            if type.isVariable {
-                let bodyOffsetEnd = structure.asDictionary.bodyLength + structure.asDictionary.bodyOffset
+        array.enumerate().forEach { childNumber, structure in
```

Comment:
Could you please rename `childNumber ` to `childIndex`? It's seems more suggestive to me.